Source/AliveLibAO/TimedMine.cpp

```diff
@@ -359,4 +359,25 @@ void TimedMine::VOnThrowableHit_408B10(BaseGameObject* /*pFrom*/)
     field_114_timer = gnFrameCount_507670;
 }
 
+void TimedMine::vOnPickUpOrSlapped_408A80()
+{
+    if (field_10C_armed != 1)
+    {
+        field_10C_armed = 1;
+        if ((signed int) (field_10E_explode_timeout & 0xFFFC) >= 72)
```

Comment:
some other rounding up value?

Reply:
Most likely, although the mask is rather short, it might zero the high bits if they're occupied